plugins/Toolbox/src/Toolbox.py

```diff
@@ -126,6 +125,16 @@ def __init__(self, application: CuraApplication) -> None:
     showLicenseDialog = pyqtSignal()
     uninstallVariablesChanged = pyqtSignal()
 
+    ##  Go back to the start state (welcome screen or loading if no login required)
+    def _restart(self):
+        self._updateRequestHeader()
+        # For an Essentials build, login is mandatory
+        if not self._application.getCuraAPI().account.isLoggedIn and ApplicationMetadata.CuraBuildType == "essentials":
```

Comment:
I think there's already an isEnterprise flag in some config file somewhere right? That would be cleaner than hardcoding 'essentials' here again.

Reply:
That has yet to be merged, but it would be nice to use the same code eventually.

Reply:
There is no such flag. It's stored in CuraVersion.py as either `debug`, `release` or `essentials`.

A flag has the disadvantage of requiring additional flags for each of our types, of which multiple could be true. If something symbolic would be desired we can go for an enum. But since we're working in Python and there is no pre-compilation going on there is hardly any advantage to that over strings, so people tend to call it "Pythonic" to use strings instead of enums.

Reply:
https://github.com/Ultimaker/Cura/pull/6616/files#diff-1bce51b6e513b01618a843adcc615850R48

Reply:
@Ghostkeeper I did make an 'IsSecured' flag (in the other branch), which currently is on if it is an 'essentials' build, on the groudns that we might keep the functionality separate. On second thought, that wouldn't work here then ... so nevermind :-)

Reply:
Mmm I agree it's maybe better to separate those indeed. Nvm then, was just a thought.

Reply:
Added an `IsEnterpriseVersion` flag in 689562e61